Regression/WarpX-tests.ini

```diff
@@ -2100,4 +2117,3 @@ compareParticles = 1
 particleTypes = electron proton
 analysisRoutine = Examples/analysis_default_regression.py
 tolerance = 1.e-14
-
```

Comment:
Can this change (deleting the line) be reverted since this is touching code unrelated to the context of this PR?

Reply:
No problem, I will revert this change in the next commit.